test/service/test_repository_proxy.py

```diff
@@ -134,13 +134,11 @@ def test_repository_files_proxy(self):
                 with mock.patch.object(IndexQueryService,
                                        'get_data_file',
                                        return_value=file_doc):
-                    azul_url = furl(
-                        url=self.base_url,
-                        path=['fetch' if fetch else '', 'repository', 'files', file_uuid],
-                        args={
-                            'catalog': self.catalog,
-                            'version': file_version,
-                        })
+                    azul_url = self.base_url.set(
+                        path=['repository', 'files', file_uuid],
+                        args=dict(catalog=self.catalog, version=file_version))
```

Comment:
https://github.com/DataBiosphere/azul/blob/develop/CONTRIBUTING.rst#14line-wrapping-and-indentation
```suggestion
                    azul_url = self.base_url.set(path=['repository', 'files', file_uuid],
                                                 args=dict(catalog=self.catalog, version=file_version))
```